control-plane-operator/controllers/hostedcontrolplane/nto/clusternodetuningoperator.go

```diff
@@ -219,6 +219,8 @@ func ReconcileDeployment(dep *appsv1.Deployment, params Params) error {
 		{Name: "KUBECONFIG", Value: "/etc/kubernetes/kubeconfig"},
 	}...)
 
+	const apiServiceCertName = "apiservice-cert"
```

Comment:
Should this be moved with the other const at the top of the file?

Reply:
Sure.
Done
Thanks for the comment